src/common/RootProviders.tsx

```diff
@@ -12,7 +12,8 @@ import { ActionSheetProvider } from '@expo/react-native-action-sheet';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 import { ThemedNavigationContainer } from './ThemedNavigationContainer';
 import { LoggedInProviders } from './LoggedInProviders';
-import { PersistedQueryProvider } from '../hooks/PersistedQueryProvider';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { QueryClient } from '@tanstack/query-core';
```

Comment:
The auto-complete imports have been behaving oddly for me, so I'm not surprised this happened. But, we should import `QueryClient` from `@tanstack/react-query`, not `query-core`.